ui/backend/dbservice/getplgnstats.go

```diff
@@ -0,0 +1,42 @@
+package dbservice
+
+import (
+	"strconv"
+
+	hookDbService "github.com/aquasecurity/postee/dbservice"
+	bolt "go.etcd.io/bbolt"
+)
+
+func GetPlgnStats() (r map[string]int, err error) {
```

Comment:
There are not locks for getting stat info, is it right?

Reply:
yes.
In worth case stats won't be shown.